Shopify/cli
# Fix deploying extensions with capabilities

### WHY are these changes introduced?
We were sending the capabilities of an extension in camelCase but the server expects them in snake_case

Fixes https://github.com/Shopify/cli/issues/557



### WHAT is this pull request doing?
- Directly parse the capabilities in `snake_case` instead of transforming them to `camelCase`



### How to test your changes?
- Have an app with a `checkout ui` extension
- Modify the toml of that extension and add capabilities like:
```
[capabilities]
network_access = true
```
- try to deploy, it should work without any validation error



### Post-release steps



### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
